using-mobile-extensions/mobile-core/signal/signals-api-reference.md

```diff
@@ -0,0 +1,4 @@
+# Signal API reference
+
+
```

Comment:
Maybe we can move the extensionVersion API here and also mention that this extension listens for the information provided through collectPII API and add a link here.